pkg/services/ngalert/notifier/alertmanager_test.go

```diff
@@ -54,6 +54,7 @@ func setupAMTest(t *testing.T) *Alertmanager {
 }
 
 func TestPutAlert(t *testing.T) {
+	t.SkipNow()
```

Comment:
Could you explain why this no longer passes? Or are there just outdated references?

Reply:
This test is asserting on the Alertmanager internals. It uses what the Alertmanager calls the "[marker](https://github.com/prometheus/alertmanager/blob/main/types/types.go#L93)" to assert for alerts that have been received but not processed. 

These components are private and are no longer accessible in any form from `grafana/alerting` (as a consumer shouldn't have a need to access them).

The test was ported 1:1 to grafana/alerting in https://github.com/grafana/alerting/blob/main/alerting/grafana_alertmanager_test.go#L34 so I reckon is safe to skip for now. There are several integration tests that asserts this functionality anyway.